6-AdvancedScenarios/4-hybrid-SPA/App/package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "hybrid-spa-app",
```

```suggestion
  "name": "hybrid-spa-app",
```

let's follow the naming convention used in the rest of the samples

---

Done!